pkg/utils/utils.go

```diff
@@ -112,7 +124,16 @@ func DiscoverSriovDevices() ([]sriovnetworkv1.InterfaceExt, error) {
 				}
 			}
 		}
-		pfList = append(pfList, iface)
+
+		for _, devName := range deviceNames {
+			for _, nic := range net.NICs {
```

Comment:
@SchSeba If we only loop against network devices here, will N3000 network function still be discovered?
I guess this is the same as asking if N3000 network function will be discovered as a separate PCI address.

Reply:
Sorry for the late response

I check it and it will work also on the N3000 card

```
88:00.0 Ethernet controller: Intel Corporation Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking (rev 02)
88:00.1 Ethernet controller: Intel Corporation Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking (rev 02)
89:00.0 Processing accelerators: Intel Corporation Device 0b30
8a:00.0 Ethernet controller: Intel Corporation Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking (rev 02)
8a:00.1 Ethernet controller: Intel Corporation Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking (rev 02)

[root@cnfdb5 core]# ls /sys/bus/pci/devices/0000\:88\:00.0/net -la
total 0
drwxr-xr-x. 3 root root 0 Jan 17 15:53 .
drwxr-xr-x. 6 root root 0 Jan 17 15:53 ..
drwxr-xr-x. 5 root root 0 Jan 17 15:53 ens5f0
[root@cnfdb5 core]# ls /sys/bus/pci/devices/0000\:88\:00.1/net -la
total 0
drwxr-xr-x. 3 root root 0 Jan 17 15:53 .
drwxr-xr-x. 6 root root 0 Jan 17 15:53 ..
drwxr-xr-x. 5 root root 0 Jan 17 15:53 eth5
[root@cnfdb5 core]# ls /sys/bus/pci/devices/0000\:8a\:00.0/net -la
total 0
drwxr-xr-x. 3 root root 0 Jan 17 15:53 .
drwxr-xr-x. 6 root root 0 Jan 17 15:53 ..
drwxr-xr-x. 5 root root 0 Jan 17 15:53 eth6
[root@cnfdb5 core]# ls /sys/bus/pci/devices/0000\:8a\:00.1/net -la
total 0
drwxr-xr-x. 3 root root 0 Jan 17 15:53 .
drwxr-xr-x. 6 root root 0 Jan 17 15:53 ..
drwxr-xr-x. 5 root root 0 Jan 17 15:53 ens5f1
```